streams/core.py

```diff
@@ -105,6 +105,23 @@ def emit(self, x):
                 result.append(r)
         return [element for element in result if element is not None]
 
+    def update(self, x, who=None):
+        self.emit(x)
+
+    def connect(self, parent):
+        ''' Connect another child to stream.
+            Note that parents go downstream and children go upstream.
```

Comment:
The numpydoc standard recommends having a single-line description, followed by a blank line, followed by a more extended description if necessary.  https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections

http://dask.pydata.org/en/latest/develop.html#docstrings